packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -217,6 +244,8 @@ export const guessFieldTypes = (series: DataFrame): DataFrame => {
 
 export const isTableData = (data: any): data is DataFrame => data && data.hasOwnProperty('columns');
 
+export const isJSONDocumentData = (data: any): data is DataFrame => data && data.meta && data.meta.json;
```

Comment:
do we need this in `@grafana/data`?  can it be isolated to just the table panel?  I don't want to encourage that this gets used anywhere new

Reply:
Sure. WIll fix